plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/messages.properties

```diff
@@ -4,8 +4,8 @@ LOGIN_ERROR_CANNOT_OPEN_BROWSER=Error opening a system browser. Signing in requi
 LOGIN_ERROR_DIALOG_MESSAGE=An error occurred while trying to sign in.
 LOGIN_ERROR_DIALOG_TITLE=Sign in Failed
 LOGIN_ERROR_LOCAL_SERVER_RUN=Error listening for authorization code:\n\n{0}
-LOGIN_MENU_LOGGED_IN=Manage Google Accounts...
-LOGIN_MENU_LOGGED_OUT=Sign in to Google...
+LOGIN_MENU_LOGGED_IN=View Application Default Credentials account...
+LOGIN_MENU_LOGGED_OUT=Please log in via gcloud CLI...
```

Comment:
If a new user see this, it seems reasonable they'd have one of two thoughts:
* I don't know how to do that.
* I should go execute `gcloud auth login`

But neither of these is the action we're looking to have completed.

Rather than fitting the request in the disabled menu item, do we need to open the dialog and display more information about specifically what we're asking the user to do?

Reply:
Makes sense - I will use a separate dialog